validate_release.py

```diff
@@ -342,6 +342,8 @@ def ParseDeps(dpk, out):
 
 def AnalyzeDpk(dpk, unv, symids, deps):
     dpk = zipfile.ZipFile(dpk)
+    if 'dirty' in dpk.filename:
```

Comment:
How about checking if it ends with `-dirty.dpk` instead?

Reply:
Tightened this to check only the version part for `-dirty`.